quipucords/scanner/satellite/six.py

```diff
@@ -139,22 +139,38 @@
 }
 
 
-# pylint: disable=too-many-locals,too-many-statements,too-many-branches
-def host_fields(scan_task, api_version, url, org_id, host_id):
+# pylint: disable=too-many-locals,too-many-statements
+# pylint: disable=too-many-branches,too-many-arguments
+def host_fields(scan_task, api_version, url, org_id, host_id,
+                ssl_cert_verify=None, host=None, port=None, user=None,
+                password=None):
     """Obtain the fields for a given host id.
 
     :param scan_task: The scan task being executed
     :param api_version: The version of the Satellite api
     :param url: The endpoint URL to get data from
     :param org_id: The organization identifier
     :param host_id: The identifier of the host being queried.
+    :param ssl_cert_verify: The value defined for ssl_cert_verify
```

Comment:
Maybe we should group these new params into an options dict.  Can be next PR.

Reply:
This is a great idea! I will do that in #1455